### PR TITLE
Add blocking-comment to all-repos.yaml

### PR DESCRIPTION
**Pull request by dbhagen**
Add `blocking-comment`: https://github.com/dbhagen/blocking-comment

my new repository:

- [X] is added to the bottom *or* with existing repos from the same account
- [X] contains a license
- [X] is not `language: system`, `language: script`, `language: docker`, or `language: docker_image`
- [X] does not contain "pre-commit" in the name

### CODE REVIEW

**Comment by dbhagen** [comment]
Didn't know `fail_fast` wasn't allowed, removed.

File: all-repos.yaml [hunk 1]
```diff
@@ -247,3 +247,4 @@
 - https://github.com/hhatto/autopep8
 - https://github.com/rhysd/actionlint
 - https://github.com/hija/clean-dotenv
+- https://github.com/dbhagen/blocking-comment
```
Comment (asottile) [inline all-repos.yaml:250]: this can be entirely replaced with a no code `pygrep` pattern

Comment (dbhagen) [reply all-repos.yaml:250]: I don't disagree, that's how I started, but having a specific hook that displays a specific message on error was more applicable to my needs. If that doesn't fit the community contributed hooks, that's cool, just something I use this specific way and figured others might be interested in.

Comment (dbhagen) [reply all-repos.yaml:250]: Feel free to merge in or close the PR, either way.

**Pull request closed without merging**